src/jit.rs

```diff
@@ -458,3 +464,10 @@ fn declare_variable(
     }
     var
 }
+
+// Prints a value used by the compiled code. Our JIT exposes this
+// function to compiled code with the name "print".
+fn print_internal(value: isize) -> isize {
```

Comment:
```suggestion
extern "C" fn print_internal(value: isize) -> isize {
```
Otherwise rustc may use any calling convention it wants including one incompatible with the call conv used by the jotted code.

Reply:
Ah nice. Thanks for the feedback.